### PR TITLE
updating gnome note

### PR DESCRIPTION
**Pull request by anaxceron**
Describe your pull request here: Addressing [Issue 3837](https://github.com/zowe/docs-site/issues/3837)

List the file(s) included in this PR: cli-configure-scs-on-headless-linux-os.md

After creating the PR, follow the instructions in the comments.

### CODE REVIEW

File: docs/user-guide/cli-configure-scs-on-headless-linux-os.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ gnome-keyring-daemon -r --unlock --components=secrets
 
 :::note
 
-The `gnome-keyring-daemon -r --unlock --components=secrets` command prompts you to specify a password. After you enter the password, press `Ctrl` + `D` twice to continue the terminal session.
+The `gnome-keyring-daemon -r --unlock --components=secrets` command can appear to delay, but it is waiting for you to enter a password. Type your keyring password (typically the same as your Linux user password), then press `Ctrl`+`D` twice to continue the terminal session.
```
Comment (t1m0thyj) [inline docs/user-guide/cli-configure-scs-on-headless-linux-os.md:34]: Would "hang" be better than "delay" here? I think a delay is slow/late response, but what we're experiencing here is an indefinite hang up that will never return unless the user aborts the command.

Comment (anaxceron) [reply docs/user-guide/cli-configure-scs-on-headless-linux-os.md:34]: @t1m0thyj: Changing this to "hang" -- wasn't aware of this definition, but now that I am, I agree w/ this. Thank ya!

**Review by t1m0thyj — APPROVED** [review]
LGTM, thanks @anaxceron!

**Review by adam-wolfe — APPROVED** [review]
LGTM, thanks @anaxceron

**Pull request merged**